argocd/overlays/moc-infra/configs/argo_rbac_cm/policy.csv

```diff
@@ -30,3 +30,5 @@ g, sre, role:standard-user
 g, fybrik, role:standard-user
 g, tremor-demo, role:standard-user
 g, osc-admins, role:standard-user
+g, peribolos-as-a-service, role:standard-user
```

Comment:
I think there was some reason to *not* include the EOL at EOF here. cc @HumairAK for confirmation.

Reply:
yeah there was...but the reason is escaping me, so just be safe and keep it without eol